Kernel/VM/PhysicalRegion.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <AK/Retained.h>
+#include <Kernel/Assertions.h>
+#include <Kernel/PhysicalAddress.h>
+#include <Kernel/VM/PhysicalPage.h>
+
+class PhysicalRegion : public Retainable<PhysicalRegion> {
+    AK_MAKE_ETERNAL
+
+    friend class MemoryManager;
```

Comment:
Is this friendship needed?

Reply:
At the moment I think it is - `PhysicalRegion` modifies `s_user_physical_pages_not_yet_used`, which is owned by `MemoryManager`. The intention there was to (cheaply) let `MemoryManager` know about how much unused memory exists. If there's a better way to do it without poking into any private data, I'd be more than happy to change it!

Reply:
This is no longer relevant :D